### ISSUE TITLE
Apply Bless in a compass project

### ISSUE THREAD

**Issue by jmmerino** [bdkjones/CodeKit-1-Old#163]
There is no way to run bless in css that is managed by compass. I apreciate that feature.

**Comment by jfexyz** [comment]
+1
This is a crucial feature, as all my projects use Compass and therefore can't benefit from Bless. But unfortunately I have to use Bless, so can't use CodeKit on these projects...

**Comment by GianlucaGuarini** [comment]
I had the same problem but I've found a solution on stakoverflow http://stackoverflow.com/questions/13259419/how-to-combine-compass-with-bless hoping that it could be useful also for you guys

**Comment by jfexyz** [comment]
Yep, this is the same solution I went with a while back. But it would still be nice to round out CodeKit's native Bless support for all preprocessors...

**Comment by bdkjones** [comment]
It's on my list of things to tackle.
